metadata/commit_classes.json

```diff
@@ -53,6 +53,17 @@
 			"^[Oo]riginally[- ]landed[- ]as:"
 		]
 	},
+	{
+		"name": "Test-import",
+		"pickable": false,
+		"headers": ["[Ii]mport", "[Ss]ync"],
+		"paths": [
+			"LayoutTests/TestExpectations",
+			"LayoutTests/tests-options.json",
+			"LayoutTests/platform",
```

Comment:
Can we use wildcards? Because it would be kinda nice to have this be `LayoutTests/platform/*/imported`.

Reply:
We can, as it turns out! Let me modify this....